app/services/port.js

```diff
@@ -9,53 +10,49 @@ export default Service.extend(Evented, {
     this._super(...arguments);
 
     /*
-     * An array of objects of the form:
-     * { applicationId, applicationName }
+     * A dictionary of the form:
+     * { applicationId: applicationName }
```

Comment:
Keep track of the names as well, in addition to the IDs. Changed from an array to a dictionary object.